main.py

```diff
@@ -22,129 +42,149 @@ def main():
         # In the above example, 'main.py' is argument 0, and the path is argument 1.
         # This is because all indexing in programming languages starts at 0.
 
-        # If the number of arguments passed is greater than or equal to 2...
-        if len(sys.argv) >= 2:
-                # Set the path variable to the 2nd argument (path).
-                path = sys.argv[1]
+        if len(sys.argv) < 2:
+            print('Please call the script with the path to a video file!\n'
+                  'e.g. python3 main.py path/to/video.mkv')
```

Comment:
For some reason I kept accidentally running the script without passing the video file (lol), so I thought this would be handy. 😊